tools/packaging/common/istio-start.sh

```diff
@@ -99,10 +99,6 @@ if [ "${ISTIO_INBOUND_INTERCEPTION_MODE}" = "TPROXY" ] ; then
   EXEC_USER=root
 fi
 
-if [ -z "${PILOT_ADDRESS:-}" ]; then
```

Comment:
I am not sure its safe to always remove this, as user may customize it. Maybe we can only not set it if ISTIO_SYSTEM_NAMESPACE and ISTIO_PILOT_PORT are unset?